prep-html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <h1 style="color: orange">
+    My favorite Star Wars character is Wedge Antilles
+  </h1>
+  <img
+    src="https://static1.cbrimages.com/wordpress/wp-content/uploads/2021/07/Wedge-Antilles-Star-Wars.jpg"
+    alt="Wedge Antilles" />
+  <p style="color: blue">
+    He is the best pilot in the SW universe and is not even force sensitive.
+  </p>
```

Comment:
Hi Anthony, be sure to put the visible content of your page between `<body></body>` tags

Reply:
![Screenshot 2021-11-07 140827](https://user-images.githubusercontent.com/93688727/140663675-7b1d30f0-e060-4d89-96bc-e30661b7f443.jpg)

Hello, i have made the updates and added the body tags.